modules/installation-guide/partials/proc_configuring-communication-between-che-components.adoc

```diff
@@ -0,0 +1,62 @@
+[id="configuring-communication-between-{prod-id-short}-components_{context}"]
+= Configuring communication between {prod-short} components
+
+To communicate together, {prod-short} components use the internal services names, which are exposed in the internal {platforms-name} network.
+
+The administration can disable the use of the internal services names to force {prod-short} components to use external {platforms-ingress}. Using the external {platforms-ingress} might slow the traffic and lead to issues because it uses proxies, certificates, and firewalls.
+
+Using the external {platforms-ingress} is necessary in the following situations:
+
+* To deploy {prod-short} across multiple {platforms-name} clusters. The internal services names are not shared between distinct clusters.
+* To deploy {prod-short} across multiple namespaces. The {orch-name} NetworkPolicies restricts communication between namespaces.
+* To deploy {prod-short} with the multitenant network plugin.
+
+.Prerequisites
+
+* An instance of {prod-short} running on {platforms-name}.
+* You have installed {prod-short} by using the Operator.
+
+.Procedure
+
+* Configure the `__<property-value>__` value of the `disableInternalClusterSVCNames` property in the CheCluster Custom Resource server settings. Set the value of `__<property-value>__` to:
+[horizontal]
+`true`:: To use external {platforms-ingress}.
+`false`:: To use internal {orch-name} DNS names.
+
++
+====
+[source,yaml,subs="+quotes"]
+----
+apiVersion: org.eclipse.che/v1
+kind: CheCluster
+# ...
+spec:
+  server:
+    # ...
+    disableInternalClusterSVCNames: __<property-value>__
+----
+====
+
+.Verification steps
+. Specify {prod-short} as the default project:
++
+[subs="+quotes,attributes"]
+----
+$ {orch-cli} project {prod-namespace}
+----
+. Inspect the ConfigMap properties to determine, which communication method {prod-short} uses. Run the following commands and take note of the output:
++
+[subs="+quotes,attributes,macros"]
+----
+$ {orch-cli} get configmap che -o \ 
+jsonpath='{.data.pass:[CHE_KEYCLOAK_AUTH__INTERNAL__SERVER__URL]}'
+$ {orch-cli} get configmap che -o \ 
+jsonpath='{.data.pass:[CHE_WORKSPACE_PLUGIN__REGISTRY__INTERNAL__URL]}'
+----
+* If {prod-short} components communicate externally, the output is empty.
+* If {prod-short} components communicate internally, the output is following:
++
+----
+http://keycloak.eclipse-che.svc:8080/auth
+http://plugin-registry.eclipse-che.svc:8080/v3
+----
```

Comment:
Add line:
Otherwise, if the components communicate externally, the output is empty.